### ISSUE TITLE
Confusing perhaps unnecessary constraint on metadata files

### ISSUE THREAD

**Issue by JeniT** [w3c/csvw#556]
From @chaals, re http://w3c.github.io/csvw/syntax/#overriding-metadata (see #552)

> The end of Section 5.1 has
> 
> > If the user selects existing metadata files, implementations must not use metadata located through the link header (as described in section 5.3 Link Header), file-specific metadata (as described in section 5.4 Standard File Metadata) or directory-specific metadata (as described insection 5.5 Standard Directory Metadata).
> 
> This is a bit unclear. Does it mean that if a user somehow refers to a single metadata file provided as part of the source, as part of some option for processing implementations must not use any others? If that interpretation is correct it seems overly constrictive. It should be reasonable to refer to a given file as something to use in addition to others, or in place of a specified set of other sources.

**Comment by JeniT** [comment]
I think the suggestion here is to return to the model that we had where metadata files from multiple locations were merged to provide the final metadata. We buried that approach with #511 and I don't think we want to revisit it.

**Comment by iherman** [comment]
> On 14 May 2015, at 22:01 , Jeni Tennison notifications@github.com wrote:
> 
> I think the suggestion here is to return to the model that we had where metadata files from multiple locations were merged to provide the final metadata. We buried that approach with #511 and I don't think we want to revisit it.

+1

> —
> Reply to this email directly or view it on GitHub.

---

Ivan Herman, W3C
Digital Publishing Activity Lead
Home: http://www.w3.org/People/Ivan/
mobile: +31-641044153
ORCID ID: http://orcid.org/0000-0003-0782-2704

**Comment by chaals** [comment]
Half the suggestion is to clarify the text please.

Having read through #511 it seems that the is indeed that if the user specifies metadata, no other metadata will be read in even if linked - so the user should take care to specify complete metadata, e.g. copying in linked stuff they still want applied.

I can live with that.
